src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/finalclass/Example1.java

```diff
@@ -0,0 +1,48 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="FinalClass"/>
+  </module>
+</module>
+*/
+
+// xdoc section -- start
+final class Example1 { // OK
```

Comment:
General comment: dealing with this indentation (different from the rest of the project) is annoying.

Reply:
I would like to convert our code base to be +2 instead of +4 for indentation.

Reply:
>  General comment: dealing with this indentation (different from the rest of the project) is annoying.

Yes. For me, sometimes IntelliJ is aware of what's happening(one file having 4, example having 2) but not always.